test/pkg/environment/common/setup.go

```diff
@@ -233,28 +259,26 @@ func (env *Environment) CleanupObjects(cleanableObjects []functional.Pair[client
 	Expect(env.Client.List(env, namespaces)).To(Succeed())
 	wg := sync.WaitGroup{}
 	for _, p := range cleanableObjects {
-		if !lo.ContainsBy(options.IgnoreCleanupFor, func(o client.Object) bool { return reflect.TypeOf(o) == reflect.TypeOf(p.First) }) {
```

Comment:
Seems odd that we'd do this in the first place. Are you aware if we were doing any of this? If so, might want to keep this logic in just for good measure.

Reply:
Wasn't used, was intending to use it but then found a different way